.github/workflows/test.yaml

```diff
@@ -1,11 +1,11 @@
-name: Run jest tests
+name: Run vitest tests
```

Comment:
```suggestion
name: Run unit tests
```

Personally I think it's useful to specify what kind of tests these are, rather than the specific tool we use to run them

Reply:
Totally!